pkg/kubelet/metrics/metrics.go

```diff
@@ -202,7 +202,7 @@ var (
 			Subsystem:      KubeletSubsystem,
 			Name:           RuntimeOperationsDurationKey,
 			Help:           "Duration in seconds of runtime operations. Broken down by operation type.",
-			Buckets:        metrics.DefBuckets,
+			Buckets:        []float64{.005, .01, .025, .05, .1, .25, .5, 1, 2.5, 5, 10, 60, 300, 600, 900, 1200},
```

Comment:
Would it be clearer (and look less arbitrary) if we just use exponential buckets here? i.e. 
```
...
Buckets : metrics.ExponentialBuckets(.005, 2, 18),
...
```
to make a list of `.005, .01, .025, .05, .1, .25, .5, 1, 2.5, 5, 10, 20, 40, 80, 160, 320, 640, 1280`

Reply:
Yeah, works for me, too, I don't have a very strong opinion on the exact bucket sizes, I only want a set of bigger buckets to exist

Reply:
@alvaroaleman: +1 to changing this PR to metrics.ExponentialBuckets(.005, 2, 18)

Reply:
updated, ptal